joern-cli/frontends/rubysrc2cpg/src/test/scala/io/joern/rubysrc2cpg/passes/ast/SimpleAstCreationPassTest.scala

```diff
@@ -1176,6 +1176,15 @@ class SimpleAstCreationPassTest extends RubyCode2CpgFixture {
     assocOperator.lineNumber shouldBe Some(4)
   }
 
+  "Double-quoted string literals containing \\u character" in {
```

Comment:
Can we also make sure the RHS of the assignment (i.e. the string literal is in there)?

Maybe just something along the lines of

```scala
val List(literalNode) = cpg.literal.l
literalNode.code shouldBe ???
```

Reply:
Covered here: [ff1cbd1](https://github.com/joernio/joern/pull/3371/commits/ff1cbd1c5bd36ae178029e577d5f9caf534d2a61)